build/lib/release.sh

```diff
@@ -358,53 +358,64 @@ function kube::release::create_docker_images_for_server() {
         docker_build_opts='--pull'
     fi
 
+    "${DOCKER[@]}" run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
we normally:
- use qemu scripts from third_party
- pin specific digest

we should not be pulling a third party image for this, especially with a floating `:latest`

Reply:
it's also not great to do this every time, since this is machine global and clobbering these should flake stuff, ideally we should only setup qemu + buildx if it's not currently setup (we can sniff if the current builder has multi-arch as a proxy)

see: hack/build/init-buildx.sh in the sigs.k8s.io/kind repo for a rough approach (could do with improving).

Reply:
I updated it to a pined version of qemu and now qemu and builder is initialized once. PTAL.

Reply:
I think https://github.com/kubernetes/kubernetes/pull/99080 will take care of initilizing qemu + buildx.

Reply:
Discussed this offline:
https://github.com/kubernetes/kubernetes/pull/96134#discussion_r580704333